analyze-documents-ai-docintelligence/workflow.json

```
@@ -1,39 +1,39 @@
+        "Analyze_Document_for_Prebuilt_or_Custom_models_(v4.x_API)": {
+            "type": "ApiConnection",
+            "description": "You can provide the file content of the image or URL as provided in the action. The document/Image can also be pulled from non-rest data sources using other Logic Apps connectors and actions.",
             "inputs": {
-                "headers": {
-                    "inputFileUrl": "https://raw.githubusercontent.com/Azure-Samples/cognitive-services-REST-api-samples/master/curl/form-recognizer/rest-api/layout.png"
-                },
                 "host": {
                     "connection": {
                         "referenceName": "formrecognizer_#workflowname#"
                     }
                 },
                 "method": "post",
-                "path": "/documentintelligence/documentModels/@{encodeURIComponent('prebuilt-read')}:analyze",
+                "headers": {
+                    "inputFileUrl": "https://raw.githubusercontent.com/Azure-Samples/cognitive-services-REST-api-samples/master/curl/form-recognizer/rest-api/layout.png"
+                },
+                "path": "/connectorinternaldocumentintelligence/documentModels/@{encodeURIComponent('prebuilt-layout')}:analyze",
                 "queries": {
-                    "api-version": "2024-02-29-preview"
+                    "api-version": "2024-11-30"
                 }
             },
-            "runAfter": {},
-            "type": "ApiConnection"
+            "runAfter": {}
         },
-        "Azure_OpenAI_-_Get_chat_completions": {
-            "description": "Update the Messages array with your own system prompts and user questions",
+        "Get_chat_completions": {
+            "type": "ServiceProvider",
             "inputs": {
                 "parameters": {
-                    "deploymentId": "@parameters('OpenAI_TextEmbedding_Deployment_Identifier_#workflowname#')",
+                    "deploymentId": "@parameters('OpenAI_TextEmbedding_Deployment_Identifier_docIntelligence",
                     "messages": [
                         {
-                            "content": "You are an AI assistant that answers questions. You will be provided markdown content based on which you can answer the questions",
-                            "role": "System"
+                            "role": "System",
+                            "content": "You are an AI assistant that answers questions. You will be provided markdown content based on which you can answer the questions"
                         },
                         {
-                            "content": "Here are the questions - 1. What is the confirmation ID and booking date.  2.) What is estimated time of departure by tone late spoken.  3. What is the Name and email on booking Here is markdown - @{body('Analyze_Document_for_Prebuilt_or_Custom_models_(v4.x_API)_1')?['analyzeResult']?['content']}",
```

- Can't add a comment here, so I'll leave this suggestion.
- What does "tone late spoken" mean?

```suggestion
"content": "Here are the questions: \n- What are the confirmation ID and booking date? \n- What is the estimated departure time by tone late spoken? \n- What are the name and email on the booking? \n- Here is the Markdown: '@{body('Analyze_Document_for_Prebuilt_or_Custom_models_(v4.x_API)_1')?['analyzeResult']?['content']}'",
```

---

@divyaswarnkar

---

@ecfan @Elaina-Lee this is based on the document here - https://raw.githubusercontent.com/Azure-Samples/cognitive-services-REST-api-samples/master/curl/form-recognizer/rest-api/layout.png

It has a table and that's the value in By column

---

The example doesn't really make much sense. Does the By column contain a courier or delivery service name?

---

yes its a delivery service name I believe. It's sample document from Document Intellignce team's page. https://github.com/Azure-Samples/document-intelligence-code-samples/blob/main/Data/layout/layout.png

---

@ecfan let me know if we can resolve this and merge PR